public/app/core/components/NavBar/Next/NavBarMenu.tsx

```diff
@@ -32,6 +33,12 @@ export function NavBarMenu({ activeItem, navItems, onClose }: Props) {
     <div data-testid="navbarmenu" className={styles.container}>
       <FocusScope contain restoreFocus autoFocus>
         <nav className={styles.content} ref={ref} {...overlayProps} {...dialogProps}>
+          <IconButton
+            name={isOpen ? 'angle-left' : 'angle-right'}
+            className={styles.menuCollapseIcon}
+            size="xl"
+            onClick={onClose}
+          />
```

Comment:
Can we combine the `IconButton` here and the chevron on `NavBarNext` into its own component? So we don't have duplicate code. I saw that the only difference is the `transform`, not sure why one has it and the other doesn't, if it is hard to combine both we can leave that for another PR where we clean the code